ci-operator/step-registry/openshift-qe/rhoso/nfv-e2e/cleanup/openshift-qe-rhoso-nfv-e2e-cleanup-commands.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+SSH_ARGS="-i ${CLUSTER_PROFILE_DIR}/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -o ServerAliveInterval=60 -o ServerAliveCountMax=240"
+jumphost=$(cat ${CLUSTER_PROFILE_DIR}/address)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)